virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -27,6 +27,8 @@ In {VirtProductName} clusters installed using installer-provisioned infrastructu
 
 * To use proxy with {VirtProductName}, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager].
 
+* When migrating virtual machines to a cluster, ensure that all nodes in a single pool use the same type of CPU. For example, nodes in one pool can use AMD processors and nodes in another pool can use Intel processors, and so on. Avoid mixing different CPU types in the same pool. 
```

Comment:
I think the tense is confusing ("When migrating..." vs. "Before you can migrate"). I also think this needs to be a bit more direct. 

(Also, I searched the repo to double check and we generally use "to" in cases like this.)

Proposed rewrite:

```suggestion
* Before you can migrate virtual machines to a cluster, you must ensure that all nodes in a single pool use the same type of CPU. Do not mix different CPU types in the same pool. For example, place nodes that use AMD processors in one pool and place nodes that use Intel processors in a different pool.
```

Reply:
The "before" method sounds good to me because this section is part of the prerequisites so it fits.
This issue is right about 2.4 -2.5 (I think Its solved on 2.6 but I will check)

Reply:
We can go with Before if that makes sense. The other prereqs in the assembly do not "before" as they are written more as general guidelines rather than sequential steps in a task. 

I do not think "you must" is needed as "ensure" is a directive verb. I also think the example should come immediately after the initial guideline. The other reviewers I consulted (Catherine and Nunzy) seem to feel the same way on these two points.

Rewriting as:
_Before you can migrate virtual machines to a cluster, ensure that all nodes in a single pool use the same type of CPU. For example, place nodes that use AMD processors in one pool and place nodes that use Intel processors in a different pool. Do not mix different CPU types in the same pool._

Reply:
Also dropping "can" --- it is not needed.

Reply:
Sounds good to me :)

Reply:
One thing , maybe we should write INTEL(all capital)  , we did write AMD , do we want to stay on the same "line" ?

Reply:
Hi @kobi86, AMD is capitalized as it is the commonly used acronym to describe Advanced Micro Devices, Intel is the  company name, that's the difference. No worries. :-) 

Should this be published for 2.6 or only for 2.4 and 2.5?

 Bob

Reply:
2.4 and 2.5 for sure 

about 2.6 - I would like to check to make sure